app/controllers/dashboards_controller.rb

```diff
@@ -0,0 +1,22 @@
+class DashboardsController < ApplicationController
+  before_action :require_login
+
+  def show
+    unless current_user.eventbrite_token?
+      @eventbrite_auth_url = eventbrite_auth_url
+    end
+  end
+
+  private
+
+  def eventbrite_auth_url
+    client ||= OAuth2::Client.new(
+      ENV.fetch('EVENTBRITE_OAUTH_ID'),
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.